src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -939,7 +971,8 @@ private void updateOrderStatus(CompositePoLine compOrderLine, JsonObject lineFro
    // See MODORDERS-218
     if (!StringUtils.equals(poLine.getReceiptStatus().value(), compOrderLine.getReceiptStatus().value())
           || !StringUtils.equals(poLine.getPaymentStatus().value(), compOrderLine.getPaymentStatus().value())) {
-      HelperUtils.sendEvent(MessageAddress.RECEIVE_ORDER_STATUS_UPDATE, createUpdateOrderMessage(compOrderLine), requestContext);
```

Comment:
I see that HelperUtils.sendEvent is using in 3 places, what do you think, what we should do with other 2 places?